app/src/main/res/values-af/strings.xml

```diff
@@ -31,7 +31,7 @@
     <string name="label_off">OFF</string>
     <string name="label_value_copied">%s copied</string>
     <string name="label_disable">Disable</string>
-    <string name="folder_lable_access">Access</string>
+    <string name="folder_lable_access">ACCESS</string>
```

Comment:
There is a typo in this string name: "lable" 😄 
Also, someone earlier in the same situation told me that it's always better to not make strings uppercase but rather uppercase them later in the code when needed and that's what we should rather stick to 😉

Reply:
:D good ideas, fix both of them.